docs/cloud/serverless.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Serverless 
+layout: default
+parent: Cloud
```

Comment:
This needs to go under Databases because that's what it's related to. This means the following YAML alteration:

```
parent: Manage databases
grand_parent: Cloud
```